### ISSUE TITLE
No matching distribution found for win32ctypes (from keyring>=3.3->jrnl) (Windows 10)

### ISSUE THREAD

**Issue by vorou** [maebert/jrnl#408]
I have a problem installing jrnl

Win10
Python 3.4.3
pip 8.0.2

Here's the full log:

```
$ pip install jrnl
Collecting jrnl
  Using cached jrnl-1.9.8.tar.gz
Collecting parsedatetime>=1.2 (from jrnl)
  Using cached parsedatetime-1.5.tar.gz
Collecting pytz>=2013b (from jrnl)
  Using cached pytz-2015.7-py2.py3-none-any.whl
Requirement already satisfied (use --upgrade to upgrade): six>=1.6.1 in c:\tools\python\lib\site-packages (from jrnl)
Collecting tzlocal>=1.1 (from jrnl)
  Using cached tzlocal-1.2.tar.gz
Collecting keyring>=3.3 (from jrnl)
  Using cached keyring-8.3.tar.gz
Requirement already satisfied (use --upgrade to upgrade): colorama>=0.2.5 in c:\tools\python\lib\site-packages (from jrnl)
Collecting python-dateutil>=2.2 (from jrnl)
  Using cached python_dateutil-2.4.2-py2.py3-none-any.whl
Collecting pyreadline>=2.0 (from jrnl)
  Using cached pyreadline-2.1.zip
Collecting win32ctypes (from keyring>=3.3->jrnl)
  Could not find a version that satisfies the requirement win32ctypes (from keyring>=3.3->jrnl) (from versions: )
No matching distribution found for win32ctypes (from keyring>=3.3->jrnl)
```

**Comment by maebert** [comment]
Can you try running `pip install keyring==7.3` before installing jrnl?

**Comment by vorou** [comment]
Yay, it works!

Just in case anybody would try to build pycrypto on Windows — you don't want to. Just download the [binaries](http://www.voidspace.org.uk/python/pycrypto-2.6.1/).

**Comment by maebert** [comment]
Haha, indeed. Issue is fixed in next release (see #407).
